src/allotropy/allotrope/models/liquid_chromatography_rec_2023_03_liquid_chromatography.py

```diff
@@ -98,6 +98,83 @@ class InjectionDocument:
     injection_time: TDateTimeValue
 
 
+@dataclass
```

Comment:
This item was previously manually created because the model generation script could not handle it, now it is automatically generated.